### ISSUE TITLE
rule window is too wide

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [AlexRiina/anki_cousins#23]
Hi,

currently the window has button that are as wide as the longest name in your notetypes. This make my window larger than my vertical screen even though I rarely use this notetype anyway.

See the picture : 
![image](https://user-images.githubusercontent.com/26625900/117215460-c4783a00-aded-11eb-96cb-eee3b7bb61b2.png)

all the space in the rectangle is waster IMO

**Comment by AlexRiina** [comment]
good point. I left the widgets pretty much unconfigured, but could definitely look into shrinking those fields. Hopefully pyqt has some kind of widget that doesn't take up so much space because I'd rather not truncate the text myself.
